addons/web/static/src/fields/integer_field.js

```diff
@@ -41,7 +41,7 @@ IntegerField.defaultProps = {
     inputType: "text",
     setAsInvalid: () => {},
 };
-IntegerField.isEmpty = () => false;
+IntegerField.isEmpty = (record, fieldName) => record.data[fieldName] === false ? true : false;
```

Comment:
Don't we default numeric field values to 0?

Reply:
yes, but samething as char, a numeric value can be unset